Mathlib/Topology/Algebra/Module/WeakDual.lean

```diff
@@ -314,6 +314,45 @@ instance instTopologicalSpace : TopologicalSpace (WeakSpace 𝕜 E) :=
 instance instContinuousAdd : ContinuousAdd (WeakSpace 𝕜 E) :=
   WeakBilin.instContinuousAdd (topDualPairing 𝕜 E).flip
 
+
+/-- There is a canonical map `E → WeakSpace 𝕜 E` (the "identity"
+mapping). It is a linear equivalence. -/
+def toWeakSpace : E ≃ₗ[𝕜] WeakSpace 𝕜 E := LinearEquiv.refl 𝕜 E
+-- align?
+
+@[simp]
+-- theorem coe_toWeakSpace (x : E) : (toWeakSpace x : WeakSpace 𝕜 E) = x :=
+theorem coe_toWeakSpace (x : E) : toWeakSpace x = x :=
+  rfl
+-- #align ?
```

Comment:
This theorem should not exist since it does the dreaded def-eq abuse.

Reply:
ok, does it mean that [WeakDual.coe_toNormedDual](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/NormedSpace/WeakDual.html#WeakDual.coe_toNormedDual) should not exist as well?

Reply:
yes, see the other comment.

Reply:
ok, I leave `NormedDual` to you (or others). I deleted this theorem.